src/telemetry/telemetry.ts

```diff
@@ -112,11 +92,34 @@ class Telemetry implements ITelemetry {
 
   private includeSpecifiedTelemetryTypes(envelope: ITelemetryItem) {
     const baseType = envelope.baseType || '';
-    const typesToInclude = ['EventData', 'MetricData', 'ExceptionData', 'PageviewData'];
-    if (typesToInclude.includes(baseType)) {
+    const telemetryTypesToInclude = [
+      'EventData',
+      'MetricData',
+      'PageviewData',
+      'ExceptionData',
+      'RemoteDependencyData'
+    ];
+    return telemetryTypesToInclude.includes(baseType);
+  }
+
+  private filterRemoteDependencyData(envelope: ITelemetryItem) {
+    const baseData = envelope.baseData || {};
+    if (envelope.baseType === 'RemoteDependencyData') {
+      const targetsToInclude = {
+        GRAPHAPI: 'graph.microsoft.com',
+        DEVXAPI: 'graphexplorerapi.azurewebsites.net'
```

Comment:
These urls are present in the graph-constants.ts file. They can be called from there directly

Reply:
Sometimes `https://proxy.apisandbox.msdn.microsoft.com` when making a call as an anonymous guest. We should capture that too here